test/UnitTestGuide.md

```diff
@@ -0,0 +1,652 @@
+# **Unit Testing 101: How to Write Tests for LORIS**
+
+
+## Written: Summer 2019 for LORIS 21
+
+
+###  Alexandra Livadas, Google Summer of Code
+
+#### _[with reference to PHPUnit Manual (8.3)](https://phpunit.readthedocs.io/en/8.3/)_
+
+ 
+
+---
+
+
+This guide was written as part of Google Summer of Code.  It is for LORIS developers or anyone working with LORIS who wants to contribute unit tests to the LORIS codebase (github.com/aces/Loris) and learn best practices for setting up and writing unit tests. 
```

Comment:
Add a link to your website somewhere in the first paragraph.

Reply:
Maybe your name, a few line higher but not in this paragraph.